tokens/theme-m3-light.json

```diff
@@ -0,0 +1,621 @@
+{
+  "sys": {
+    "primary": {
+      "value": "{color.bg.action.primary.rest}",
+      "type": "color"
+    },
+    "on-primary": {
+      "value": "{color.text.on-solid.primary}",
+      "type": "color"
+    },
+    "primary-container": {
+      "value": "{color.bg.canvas.default}",
+      "type": "color"
+    },
+    "on-primary-container": {
+      "value": "{color.text.primary}",
+      "type": "color"
+    },
+    "secondary": {
+      "value": "{color.text.secondary}",
+      "type": "color"
+    },
+    "on-secondary": {
+      "value": "{color.text.on-solid.primary}",
+      "type": "color"
+    },
+    "secondary-container": {
+      "value": "{color.bg.subtle-primary}",
+      "type": "color"
+    },
+    "on-secondary-container": {
+      "value": "{color.text.primary}",
+      "type": "color"
+    },
+    "tertiary-container": {
+      "value": "{color.bg.action.primary.rest}",
+      "type": "color"
+    },
+    "on-tertiary-container": {
+      "value": "{color.text.on-solid.primary}",
+      "type": "color"
+    },
+    "error": {
+      "value": "{color.text.critical.primary}",
+      "type": "color"
+    },
+    "on-error": {
+      "value": "{color.text.on-solid.primary}",
+      "type": "color",
+      "description": "color.theme.bg"
+    },
+    "surface": {
+      "value": "{color.bg.canvas.default}",
+      "type": "color"
+    },
+    "on-surface": {
+      "value": "{color.text.primary}",
+      "type": "color"
+    },
+    "on-surface-variant": {
+      "value": "{color.text.secondary}",
+      "type": "color"
+    },
+    "surface-tint": {
+      "value": "{color.gray.1000}",
+      "type": "color",
+      "description": "Not consumed directly in components but is used to calculate the color of surface layers."
+    },
+    "inverse-surface": {
+      "value": "{color.gray.1300}",
+      "type": "color"
+    },
+    "inverse-on-surface": {
+      "value": "{color.text.on-solid.primary}",
+      "type": "color"
+    },
+    "inverse-primary": {
+      "value": "{color.text.on-solid.primary}",
+      "type": "color"
+    },
+    "outline": {
+      "value": "{color.border.interactive.primary}",
+      "type": "color"
+    },
+    "outline-variant": {
+      "value": "{color.alpha.gray.400}",
+      "type": "color"
+    },
+    "surface-container-highest": {
+      "value": "#eeeff0",
+      "type": "color",
+      "description": "Autogenerated by M3."
```

Comment:
Not sure I understand the `description`. What did autogenerated this? How do i get them to be generated again?

Reply:
The only way to get the value for those colors is to feed the tokens to Material 3, and then check what colors it generates.

We (Jorge and me) checked the source code for the function that is supposed to generate them, but its didn't match what's being output, hence the only way to know the value for those few `surface-*` tokens is to feed the Material M3 theme tokens to [ColorScheme](https://developer.android.com/reference/kotlin/androidx/compose/material3/ColorScheme) and check the output in Showkase.

Reply:
/cc @jmartinesp

Reply:
Right, could we update the `description` field to have a very condensed version of what you said above? It does make a lot of sense!